etl/helpers.py

```diff
@@ -50,3 +54,38 @@ def _get_github_branches(org: str, repo: str) -> List[Any]:
         raise Exception("reached single page limit, should paginate request")
 
     return branches
+
+
+class Naming:
+    def __init__(self, file: str):
+        self.f = Path(file)
+
+    @property
+    def channel(self) -> str:
+        return self.f.parent.parent.parent.name
+
+    @property
+    def namespace(self) -> str:
+        return self.f.parent.parent.name
+
+    @property
+    def version(self) -> str:
+        return self.f.parent.name
+
+    @property
+    def short_name(self) -> str:
+        return self.f.with_suffix("").name
+
+    @property
+    def country_mapping_path(self) -> Path:
+        return self.f.parent / (self.short_name + ".country_mapping.json")
```

Comment:
Didn't we decide to call it `.countries.json` for simplicity?

Reply:
Oh, you're right. That's exactly why I need something like this - so that I don't have to remember our conventions.